webapp/channels/src/components/analytics/activated_users_card/title.tsx

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {useIntl} from 'react-intl';
+
+import {InformationOutlineIcon} from '@mattermost/compass-icons/components';
+
+import ExternalLink from 'components/external_link';
+import OverlayTrigger from 'components/overlay_trigger';
+
+import Constants from 'utils/constants';
+
+import TitleTooltip from './title_tooltip';
+
+const Title = () => {
+    const intl = useIntl();
+    return (
+        <OverlayTrigger
+            delayShow={Constants.OVERLAY_TIME_DELAY}
+            placement='top'
+            overlay={<TitleTooltip/>}
+        >
+            <span>
+                <ExternalLink href='https://docs.mattermost.com/configure/reporting-configuration-settings.html#site-statistics'>
```

Comment:
For many of the other docs links in the app, we've linked to a permalink on the website like https://mattermost.com/pl/terms-of-use in case the layout of the docs site changes. I don't know how we get someone to set one of those up, but should we do that here?

Reply:
I guess that question is more for @cwarnermm or @johndavidlugtu . Thoughts?

Reply:
Yes good callout @hmhealey  let's use mattermost.com/pl/site-statistics-definitions

@larkox

Reply:
@johndavidlugtu `mattermost.com/pl/site-statistics-definitions` throws a page not found error. Is that expected?

Reply:
This PL link needs to be created by Marketing before it works. I'll own this step.

Reply:
Link updated. While at it, took a look at what other `docs.mattermost.com` links we have in the product and they are a lot. 0/5 if this is an effort we want to dedicate time, or a least a jira ticket.

Reply:
The PL link should now work as expected.